src/components/input/StringField.jsx

```diff
@@ -1,36 +1,56 @@
+/**
+ * Use Cases:
+ * - component receives value => string | null
+ * - user types in the input => string (maybe empty)
+ * - user clicks on the reset button => null
+ *
+ * Notes: useRef might not be needed here, as long as value={value ?? ''} does
+ * not trigger an onChange event.
```

Comment:
Just a reminder about useRef. I don't think it is needed, but can revisit this later during the `AttributeForm` PR.